lib/src/revset.rs

```diff
@@ -2580,7 +2580,7 @@ pub struct RevsetParseContext<'a> {
     pub aliases_map: &'a RevsetAliasesMap,
     pub local_variables: HashMap<&'a str, ExpressionNode<'a>>,
     pub user_email: &'a str,
-    pub date_pattern_context: DatePatternContext,
+    pub date_pattern_now: Zoned,
```

Comment:
Open to a better name.

Reply:
I would call it `current_time` or something.